.circleci/config.yml

```diff
@@ -103,3 +137,44 @@ jobs:
       - run:
           name: Build the documentation.
           command: nox -s docs
+  publish_package:
+    docker:
+      - image: python:3.7-slim
+    steps:
+      - checkout
+      - run:
+          name: Decrypt the PyPI key.
+          command: |
+            openssl aes-256-cbc -d \
```

Comment:
Note: default algorithm may change. Details here: https://github.com/googleapis/synthtool/pull/159

Reply:
Acknowledged. Will address in a follow-up.